src/test/codegen/issue-83623-SIMD-PartialEq.rs

```diff
@@ -0,0 +1,45 @@
+// This test checks that jumps generated by logical operators can be optimized away
```

Comment:
I don't know how to write codegen tests so I made this by example.
I can miss something.